build.gradle

```diff
@@ -124,7 +124,7 @@ subprojects {
 
     testCompile 'junit:junit'
     testCompile 'org.slf4j:slf4j-simple'
-
```

Comment:
Can you avoid the whitespace change and add this line back? That will avoid git conflicts.

Reply:
done

Reply:
Looks like the line is still missing, but it's just a nit.